sw.js

```diff
@@ -0,0 +1,97 @@
+/* nebraskajs - v0.1.4 - 2015-02-02
+* http://github.com/nebraskajs/nebraskajs.com/
+* Copyright (c) 2015 Zach Leatherman; MIT License */
+
+"use strict";
+importScripts('js/cache-polyfill.js');
```

Comment:
If you pull from master, I’ve added a `dist` meta tag that you can use to get the active dist directory. This will allow us to properly cache the `cache-polyfill.js` file. You can use something like this to fetch the tag content to use in the URL to the file: https://github.com/zachleat/zachleat.com/blob/master/web/js/initial.js#L3